lib/msprime.c

```diff
@@ -5090,9 +5084,9 @@ msp_run(msp_t *self, double max_time, unsigned long max_events)
     }
     if (self->store_full_arg
         && !(self->model.type == MSP_MODEL_HUDSON || self->model.type == MSP_MODEL_SMC
-               || self->model.type == MSP_MODEL_SMC_PRIME
```

Comment:
Looks like you're on the wrong version of clang-format. See [here](https://tskit.dev/tskit/docs/stable/development.html#sec-development-c-requirements) for info on getting the right one.

Yes, clang-format is awful for changing the format between versions in a non-compatible way.

Reply:
Thanks! I think that's fixed now.